### ISSUE TITLE
Browser back button don't update pages

### ISSUE THREAD

**Issue by evandrmb** [lulupointu/vrouter_website#3]
It seems that clicking on the back button isn't updating pages even tho it shows that the URL is updated as you can see below.

https://user-images.githubusercontent.com/47798920/119227931-c4af4f80-bae6-11eb-91c9-be095e0769d6.mp4

**Comment by lulupointu** [comment]
Wow it does not works only for some time and then begins to work again?
Did you do something special to trigger the issue?

**Comment by evandrmb** [comment]
Well, can't tell for sure. I don't know if this helps but like you can see I maintain a lot of tabs opened and truth be told VRouter docs was open since yesterday which made the browser keep it inactive after some time, then I started reading the docs again, and the issue appeared.

**Comment by lulupointu** [comment]
Interesting. VRouter uses `historyState` from the browser so maybe there is something with this. Did you clear you history cache or anything like this?

**Comment by evandrmb** [comment]
No, I didn't.  I'm decided to use VRouter in my projects so maybe I will have more info if this behavior happens again. Looking forward to doing a Medium post about it, because I really enjoyed all the features.

**Comment by lulupointu** [comment]
Thanks for the support!

Don't hesitate to ping me (on the discord server if you want) when it goes out 😊

Regarding the beug I tried to look into `historyState` only but it seems that this should persist as expected. I'll do some tests and let you know if I find anything,this might even be a' issue with flutter web idk.

**Comment by evandrmb** [comment]
Could you send the discord link? It seems like the link in pub dev is broken as I've tried to and Discord couldn't find the server

**Comment by lulupointu** [comment]
Oh ok that's weird: https://discord.gg/XqmgWNgr
